Rename `original_message` to `original_response` (#1609)
Pycord-Development/pycord: CHANGELOG.md

- `proxy` and `proxy_auth` params to many Webhook related methods.
  ([#1655](https://github.com/Pycord-Development/pycord/pull/1655))
- `delete_message_seconds` parameter in ban methods. ([#1557](https://github.com/Pycord-Development/pycord/pull/1557))

> - New methods `Interaction.original_response`, `Interaction.edit_original_response` & `Interaction.delete_original_response` ([#1609](https://github.com/Pycord-Development/pycord/pull/1609)

Reviewer: ```suggestion
- New methods `original_response`, `edit_original_response` & `delete_original_response` for `Interaction` objects.
  ([#1609](https://github.com/Pycord-Development/pycord/pull/1609)
```

### Deprecated
- The `delete_message_days` parameter in ban methods is now deprecated. Please use `delete_message_seconds` instead.
  ([#1557](https://github.com/Pycord-Development/pycord/pull/1557))

> - The `Interaction.original_message`, `Interaction.edit_original_message` & `Interaction.delete_original_message` methods are now deprecated. Please use `Interaction.original_response`, `Interaction.edit_original_response` & `Interaction.delete_original_response` respectively instead.

Reviewer: ```suggestion
- The `original_message`, `edit_original_message` & `delete_original_message` methods for `Interaction` are now deprecated. Please use the respective `original_response`, `edit_original_response` & `delete_original_response` methods instead.
```

- Interaction responses are now passed the respective `proxy` and `proxy_auth` params as defined in `Client`.
  ([#1655](https://github.com/Pycord-Development/pycord/pull/1655))

> ### Changed
> - Rename `original_message` to `original_response` ([#1609](https://github.com/Pycord-Development/pycord/pull/1609)
> 

Reviewer: ```suggestion
```
Please add `original_message` to the deprecated section and `original_response` to the added section.

Author: Should I add separate entries for `original_message`, `edit_original_message` and `delete_original_message`?

Reviewer: > Should I add separate entries for `original_message`, `edit_original_message` and `delete_original_message`?

no, but reference them all in the same entry
